tests/addAndRemove.test.js

```diff
@@ -0,0 +1,138 @@
+import { describe, test, expect } from '@jest/globals';
+import { addTask, deleteTask } from '../src/modules/todoFunctions.js';
+
+describe('addTask', () => {
+  test('should add a task to an empty tasks array', () => {
```

Comment:
<img width="563" alt="1" src="https://github.com/bohaz/To-do-list/assets/77843050/3ae3b51c-c731-49d7-98c8-e67326e3feb5">